pkg/api/server/v1alpha2/result/result.go

```diff
@@ -49,19 +49,21 @@ func ToStorage(r *pb.Result) (*db.Result, error) {
 		Name:        name,
 		UpdatedTime: r.UpdatedTime.AsTime(),
 		CreatedTime: r.CreatedTime.AsTime(),
+		Annotations: r.Annotations,
 	}
 	return result, nil
 }
 
 // ToAPI converts a database storage Result into its corresponding API
 // equivalent.
-func ToAPI(r *db.Result) *pb.Result {
+func ToAPI(r *db.Result) (*pb.Result, error) {
 	return &pb.Result{
 		Name:        FormatName(r.Parent, r.Name),
 		Id:          r.ID,
 		CreatedTime: timestamppb.New(r.CreatedTime),
 		UpdatedTime: timestamppb.New(r.UpdatedTime),
-	}
+		Annotations: r.Annotations,
+	}, nil
```

Comment:
We're not using error meaningfully yet - let's remove for now.

Reply:
Done :)